api/src/main/java/io/minio/MinioClient.java

```diff
@@ -4211,18 +4289,66 @@ public void disableObjectLegalHold(String bucketName, String objectName, String
    * @throws NoSuchAlgorithmException thrown to indicate missing of MD5 or SHA-256 digest library.
    * @throws XmlParserException thrown to indicate XML parsing error.
    */
+  @Deprecated
```

Comment:
It'll be good to add the deprecation javadoc comment also along with a link to the new method to be used